scss/style.scss

```diff
@@ -0,0 +1,585 @@
+  }
+
+  .author {
+    color: #fff;
+    text-align: center;
+  }
+
+  .title {
+    color: #fff;
+    text-align: center;
+    font-size: 20px;
+  }
+
+  .info {
+    font-size: 13px;
+    color: $yellow;
+
+    @include screen-size('mobile') {
+      display: none;
+    }
+  }
+
+  @include screen-size('not-mobile') {
+    position: absolute;
+    top: 70%;
+    left: 15px;
+
+    .author, .title {
+      text-align: left;
+    }
+  }
+}
+
+.button-wrapper {
+  width: 138px;
+  margin: 10px auto 0 auto;
+
+  @include screen-size('not-mobile') {
+    position: absolute;
+    top: -91px;
+    left: 200px;
+
+    .font-icon {
+      margin-right: -10px;
+    }
+  }
+
+  .font-icon {
+    cursor: pointer;
+    background-color: $yellow;
+  }
+
+  .stop-icon {
+    border-radius: 14px;
+    padding: 6px;
+  }
+
+  .play-icon {
+    border-radius: 28px;
+    padding: 14px;
+    font-size: 24px;
+    padding-left: 19px;
+  }
+
+  .forward-icon {
+    border-radius: 16px;
+    padding: 6px;
+    padding-left: 9px;
+  }
+}
+
+.recently-played {
+  .song-wrapper {
+    margin: 20px 10px;
+  }
+
+  p {
+    margin: 0 0 0 70px;
+  }
+
+  img {
+    margin-top: 5px;
+    float: left;
+  }
+
+  .song-author {
+    font-size: 15px;
+  }
+
+  .song-duration {
+    color: $grey;
+    font-weight: 700;
+  }
+}
+
+.concerts {
+  .update-date {
+    color: $grey;
+    font-size: 13px;
+  }
+
+  .concert-details {
+    font-size: 14px;
+  }
+
+  .more-button {
+    color: $yellow;
+    margin: 15px 0 0 130px;
+    width: 70px;
+
+    .more-icon {
+      border-radius: 15px;
+      border: 1px solid $yellow;
+      padding-left: 1px;
+    }
+  }
+}
+
+.contests {
+  .contest-date, .contest-details {
+    color: $grey;
+    font-size: 13px;
+  }
+}
+
+.perks {
+  .perk-date, .perk-details {
+    color: $grey;
+    font-size: 13px;
+  }
+}
+
+.links {
+  p {
+    margin: 0;
+    display: inline-block;
+  }
+
+  .more-icon {
+    float: right;
+  }
+}
+
+.stations {
+  p {
+    margin: 0;
+    display: inline-block;
+    width: 80%;
+  }
+
+  .radio-location, .radio-details {
+    color: $grey;
+    font-size: 11px;
+  }
+
+  a {
+    float: right;
+
+    .more-icon {
+      bottom: 20px;
+      position: relative;
+    }
+  }
+}
+
+@include screen-size('mobile') {
+  .currently-played {
+    height: 545px;
+  }
+
+  .mobile-background {
+    position: absolute;
+    width: 100%;
+    height: 635px;
+    background: url('img/background.png') no-repeat 100%;
+    background-size: 130%;
+  }
+}
+
+@include screen-size('tablet') {
+  .sectors {
+    position: relative;
+  }
+
+  .row {
+    width: 46%;
+    float: left;
+    display: inline-block;
+    margin: 5px;
+  }
+
+  .first-row {
+    width: 380px;
+  }
+
+  .third-row {
+    position: absolute;
+    width: 380px;
+    left: 0px;
+    top: 1100px;
+  }
+
+  .perks {
+    border: none;
+  }
+}
+
+@include screen-size('desktop') {
+  .row {
+    width: 33%;
+    float: left;
+    display: inline-block;
```

Comment:
It's a leftover, I was trying to do that with inline-block, then switched to floats and forgot to delete it.